packages/db-postgres/src/types.ts

```diff
@@ -62,7 +62,9 @@ declare module 'payload' {
     afterSchemaInit: PostgresSchemaHook[]
     beforeSchemaInit: PostgresSchemaHook[]
     beginTransaction: (options?: PgTransactionConfig) => Promise<null | number | string>
-    drizzle: PostgresDB
+    drizzle: {
+      $client: Pool
```

Comment:
If we decide to support a client connection instead of a Pool, this would be a breaking change later. I wonder if we just want to get ahead of that and do a union for both at the start.

Pretty soon v3 will not be in beta and we need to think of these things more ahead of time.

Reply:
Added